src/scripts/main.js

```diff
@@ -1,3 +1,70 @@
 'use strict';
 
-// write code here
+const appendRow = document.querySelector('.append-row');
+const removeRow = document.querySelector('.remove-row');
+const appendColumn = document.querySelector('.append-column');
+const removeColumn = document.querySelector('.remove-column');
+const table = document.querySelector('.field');
```

Comment:
It's a good practice to add a check if the element exists after querying the DOM. If the element doesn't exist, the script will crash.